md/cache-configuration-and-policy.md

```diff
@@ -1,16 +1,31 @@
-# Cache configuration and policy
+## Cache configuration and policy 
 
-This page explain how the portal cache and the living application cache are configured in bonita. You will also find 
+This page explain how the portal cache and the living application cache are configured in bonita 7.6 version. You will also find 
 details on the impact this cache configuration can have for the end user, and on how you can modify this configuration.
 
-## Portal
-
-
-Duration cache = 10h, how to change it.
-
+#### Development configuration
+ 
+ Cache policy is disabled in ui-designer preview and in bonita studio. 
```

Comment:
Cache policy is disabled by default for the web server embedded by bonita studio. So you will not have cache in development.